sys/vfs_syscalls.c

```diff
@@ -60,7 +60,7 @@ int do_lseek(thread_t *td, int fd, off_t offset, int whence) {
   /* TODO: Whence! Now we assume whence == SEEK_SET */
```

Comment:
Actually, this will be pretty difficult until #276 (Compiling newlib) is merged - because the exact values for flags `SEEK_SET`, `SEEK_CUR` and `SEEK_END` would need to match what newlib provides in it's headers.